src/Artemis.VisualScripting/Nodes/Text/StringLengthNode.cs

```diff
@@ -0,0 +1,24 @@
+﻿using Artemis.Core;
+
+namespace Artemis.VisualScripting.Nodes.Text;
+
+[Node("String Length", "Checks whether the first input is contained in the second input.", 
+    "Text", InputType = typeof(string), OutputType = typeof(Numeric))]
+public class StringLengthNode : Node
+{
+    public StringLengthNode()
+        : base("String Length", "Returns string length.")
```

Comment:
I'd rather have the display values of this node as follows
- Name: "Text length"
- Description: "Outputs the length of the input text"

I want to avoid talking about strings or return values in visual scripting, as they're technical terms

Reply:
Oh, don't forget to update line 5 as well